Modules/QuestieQuest.lua

```diff
@@ -363,7 +363,8 @@ end
 
 function QuestieQuest:CompleteQuest(questId)
     QuestiePlayer.currentQuestlog[questId] = nil;
-    Questie.db.char.complete[questId] = true --can we use some other relevant info here?
+    local quest = QuestieDB:GetQuest(questID)
```

Comment:
Instead of calling `QuestieDB:GetQuest(questID)` we could hand the `quest` from one layer above and store the Id in a local variable. So we save the call here.

Reply:
It should be cached based on the implementation of `QuestieDB:GetQuest(questID)`, but I get what you're saying. I updated it to the way you described. Let me know what you think.